internal/usecase/daemon/command_tracker.go

```
@@ -82,32 +83,32 @@ func (t *CommandRequestTracker) trackLocked(requestID, generation uint64) <-chan
 
 // Wait waits without holding the tracker or connection sender lock. Timeout
 // and cancellation remove only the matching request.
-func (t *CommandRequestTracker) Wait(ctx context.Context, clock ports.Clock, requestID, generation uint64, outcome <-chan CommandRequestOutcome) (ports.CommandResult, error) {
+func (t *CommandRequestTracker) Wait(ctx context.Context, clock ports.Clock, requestID, generation uint64, outcome <-chan CommandRequestOutcome) (protocol.CommandResult, error) {
```

golangci-lint (2.12.2)

[error] Non-inherited new context, use function like `context.WithXXX` instead

(contextcheck)